rpc_jobs/irr_role_test.yml

```diff
@@ -80,14 +80,17 @@
     image:
       - trusty:
           IMAGE: "Ubuntu 14.04 LTS (Trusty Tahr) (PVHVM)"
+          irr_flavor: "performance2-15"
     # Context is used to define the origin "release" when leap upgrading
     #  The "series" parameter is used to define the checkout within the
     #  release.
     context:
+      - kilo
       - kilo:
-          # Kilo EOL SHA
-          series: "97e3425871659881201106d3e7fd406dc5bd8ff3"
-          irr_flavor: "performance2-15"
+          series: r11.0.0
+      - kilo:
+          series: r11.1.18
```

Comment:
This will result in the same ``kilo`` job being overwritten multiple times, resulting in only one left.